src/web.go

```diff
@@ -77,72 +77,112 @@ type Web struct {
 	evaluator           *Evaluator
 }
 
+func wildcardRouteHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
Are we thinking of continuing to use an embed solution for more time? This class today is not easy to maintain. If we are planning to continue in this direction, could it be possible to move the embedded web and error handler in other go files?

Reply:
I would like to keep embedding the templates into the binary so we just have a single binary to deploy. But as we said in standup we may separate the web UI and make it talk to only the API in the future.